tests/integration/targets/ldap_search/tasks/tests/basic.yml

```
@@ -0,0 +1,19 @@
+- debug:
+    msg: Running tests/basic.yml
+
+####################################################################
+## Search ##########################################################
+####################################################################
+- name: Test simple search for a user
+  ldap_search:
+    dn: "ou=users,dc=example,dc=com"
+    scope: "onelevel"
+    filter: "(uid=ldaptest)"
+  ignore_errors: yes
+  register: output
+
+- name: assert that test LDAP user can be found
+  assert:
+    that:
+       - 'not output.failed'
+       - 'output.results.0.displayName == "LDAP Test"'
```

```suggestion
       - output.results.0.displayName == "LDAP Test"
```
you can get rid of extra quotes. IMO that makes things clearer